xla/service/gpu/cudnn_norm_rewriter_test.cc

```diff
@@ -1088,6 +1375,125 @@ TEST_F(CudnnNormRewriterTest, LayerNormTrainBackward4D12) {
   TestNorm(hlo_text, optimized_hlo);
 }
 
+// Dbias not expected to be fused into norm gradient.
```

Comment:
Comment this is because dbias is not fused when a reduction dimension is degenerate.

Also, why is Dbias not fused in this case?